puppet/modules/prprocessor/manifests/init.pp

```diff
@@ -65,6 +64,14 @@
     require     => Package[$packages],
   }
 
+  cron { 'close inactive':
```

Comment:
General question, cron is quite untrackable. Feels like whenever I see them we should be preferring something like a Jenkins job that can be monitored for failures. Thoughts?